roles/openstack-stack/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: create HOT stack template prefix
+  register: stack_template_pre
+  tempfile:
+    state: directory
+    prefix: casl-ansible
```

Comment:
We can accept this as is with the casl-ansible tag but in the future we may want to make this a configurable variable